be/src/storage/delete_handler.cpp

```diff
@@ -50,6 +51,13 @@ using google::protobuf::RepeatedPtrField;
 
 namespace starrocks {
 
+DeleteHandler::~DeleteHandler() {
```

Comment:
So, there is a memory leak here in the previous version?

Reply:
Maybe it's better to let `DeleteConditions` delete `del_cond` in its own destructor.

Reply:
yes